src/Notification/Notification.js

```diff
@@ -21,6 +21,8 @@ const Notification = ({
   variant,
   onClose,
   actions,
+  closeBtn,
```

Comment:
Better to make it clear that it's a boolean variable. Usually, we do it by prepending `is`, `has` or `with`.